serve.py

```diff
@@ -136,6 +142,29 @@ async def goto_dandiset_version(request, dataset, version):
             return response.redirect(url)
     return response.text(f"dandi:{dataset:06d}/{version} not found.", status=404)
 
+
+@app.route("/server-info", methods=["GET"])
+async def server_info(request):
+    return response.json({
+        "version": "1.0.0",
+        "cli-minimal-version": "0.5.0",
+        "cli-bad-versions": [],
+        "services": {
+            "girder": {
+                "url": GIRDER_URL,
+            },
+            "webui": {
+                "url": GUI_URL,
+            },
+            "api": {
+                "url": PUBLISH_API_URL,
+            },
+            "jupyterhub": {
+                "url": JUPYTERHUB_URL,
+            }
+        }
+    })
```

Comment:
Minor: now seeing it in action realized that it should've been indented - there is no traffic to spare

Also I wonder - why it escaped all `/` - is it needed?

```
{"version":"1.0.0","cli-minimal-version":"0.5.0","cli-bad-versions":[],"services":{"girder":{"url":"https:\/\/girder.dandiarchive.org"},"webui":{"url":"https:\/\/gui.dandiarchive.org"},"api":{"url":"https:\/\/publish.dandiarchive.org\/api"},"jupyterhub":{"url":"https:\/\/hub.dandiarchive.org"}}}
```

Reply:
JSON allows escaping of `/`, apparently in order to avoid cross-site scripting attacks somehow.